README.md

```diff
@@ -1,2 +1,2 @@
 # rayandharia.github.io
-Rayan's Personal Website
+Rayan's Personal Website WIP
```

Comment:
Nit: Why did you add WIP and not W.I.P.?

Reply:
WIP is standard acronym